src/main/java/edu/harvard/iq/dataverse/api/Files.java

```diff
@@ -818,4 +827,37 @@ public Response getExternalToolFMParams(@Context ContainerRequestContext crc, @P
     public Response getFixityAlgorithm() {
         return ok(systemConfig.getFileFixityChecksumAlgorithm().toString());
     }
+
+    @GET
+    @AuthRequired
+    @Path("{id}/guestbookResponses/count")
+    public Response getCountGuestbookResponses(@Context ContainerRequestContext crc, @PathParam("id") String dataFileId) {
+        return response(req -> {
+            DataFile dataFile = execCommand(new GetDataFileCommand(req, findDataFileOrDie(dataFileId)));
+            return ok(guestbookResponseService.getCountGuestbookResponsesByDataFileId(dataFile.getId()).toString());
+        }, getRequestUser(crc));
+    }
+
+    @GET
+    @AuthRequired
+    @Path("{id}/thumbnailClass")
```

Comment:
Moot point if we move this logic into the SPA code... But "defaultIcon" or "defaultIconType" would probably better describe what this does (?).